src/rest/__tests__/entities.test.ts

```diff
@@ -66,16 +45,110 @@ describe('Update REST entry', () => {
     },
   } as unknown as AssetProps;
 
+  const unresolvedRichTextLinks = {
```

Comment:
as the test file is going to be a little bit long, should we extract the test data into a fixture?